languages/csharp/exercises/concept/numbers-signedness/NumbersUnsignedTest.cs

```diff
@@ -0,0 +1,24 @@
+using Xunit;
+
+public class AssemblyLineTest
+{
+    [Fact]
+    public void LightSecondsFromEarthToSun() =>
+        Assert.Equal(499.0047838361564119134711521m, LightDistance.MetersToLightSeconds(meters: 149_597_870_700));
```

Comment:
Do we really want named parameters here? I feel the method name is explicit enough that this information is redundant.

Reply:
Agreed. The method is really well named and has only one parameter, so we could lose the name perhaps.

Reply:
Agreed! :)